Makefile

```diff
@@ -46,31 +48,38 @@ run: fmt vet
 
 docker-build: Dockerfile fmt vet
 	# Name the base stages so they are not lost during a cache prune.
```

Comment:
You can delete this comment.  It no longer has a meaning.

Reply:
Done